src/relay/qnn/pass/optimize.cc

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file optimize.cc
+ * \brief Sequence of QNN passes that can be called to optimize a QNN graph.
+ */
+
+#include <tvm/relay/transform.h>
+
+namespace tvm {
+namespace relay {
+
+using transform::Pass;
+
+Function QnnOptimize(const Function& f, const Map<tvm::Integer, tvm::Target>& targets) {
```

Comment:
Why not make this function as module->module?

Reply:
Good point. I will do that.